ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -180,6 +180,19 @@
          (acl/verify-subscription-management-permission
            context :update :provider-object provider-id))))))
 
+(defn check-valid-user
+  "Raise error if the user provided does not exist"
+  [context user-id]
+  (when (string/blank? user-id)
+    (errors/throw-service-error
+     :bad-request
+     "INGEST FAILED - No ID was provided. Please provide a SubscriberId or pass in a valid token."))
+  (when-not (urs/user-exists? context user-id) 
+    (errors/throw-service-error
+     :bad-request
+     (format "The user-id [%s] must correspond to a valid EDL account." 
```

Comment:
This message could also begin with `Subscription creation failed - the user-id...`